WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsFragment.kt

```diff
@@ -38,28 +39,37 @@ import javax.inject.Inject
 
 private val statsSections = listOf(INSIGHTS, DAYS, WEEKS, MONTHS, YEARS)
 
-class StatsFragment : DaggerFragment(), ScrollableViewInitializedListener {
+class StatsFragment : DaggerFragment(R.layout.stats_fragment), ScrollableViewInitializedListener {
     @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
     @Inject lateinit var uiHelpers: UiHelpers
     private lateinit var viewModel: StatsViewModel
     private lateinit var swipeToRefreshHelper: SwipeToRefreshHelper
+    private lateinit var binding: StatsFragmentBinding
     private val selectedTabListener: SelectedTabListener
         get() = SelectedTabListener(viewModel)
 
     private var restorePreviousSearch = false
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
         setHasOptionsMenu(true)
-        return inflater.inflate(R.layout.stats_fragment, container, false)
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
         val nonNullActivity = requireActivity()
+        binding = StatsFragmentBinding.bind(requireView())
```

Comment:
I believe we can use the parameter `view` here.
 
```suggestion
        binding = StatsFragmentBinding.bind(view)
```